README.md

```diff
@@ -13,153 +13,112 @@ SPDX-License-Identifier: Apache-2.0
+Shipwright can support any tool that can build container images, such as [Kaniko](https://github.com/GoogleContainerTools/kaniko),
+[Cloud Native Buildpacks](https://buildpacks.io/), and [Buildah](https://buildah.io/). See [default strategies](#build-strategies) for more information.
 
-## Custom Resources
+## Try It!
 
-Shipwright defines four CRDs:
+* We assume you already have a Kubernetes cluster (v1.18+). If you don't, you can use [Kind](/hack/install-kind.sh).
 
-* The `BuildStrategy` CRD and the `ClusterBuildStrategy` CRD is used to register a strategy.
-* The `Build` CRD is used to define a build configuration.
-* The `BuildRun` CRD is used to start the actually image build using a registered strategy.
-
-## Read the Docs
-
-| Version | Docs                           | Examples                    |
-| ------- | ------------------------------ | --------------------------- |
-| HEAD    | [Docs @ HEAD](/docs/README.md) | [Examples @ HEAD](/samples) |
-| [v0.3.0](https://github.com/shipwright-io/build/releases/tag/v0.3.0)    | [Docs @ v0.3.0](https://github.com/shipwright-io/build/tree/v0.3.0/docs) | [Examples @ v0.3.0](https://github.com/shipwright-io/build/tree/v0.3.0/samples) |
-| [v0.2.0](https://github.com/shipwright-io/build/releases/tag/v0.2.0)    | [Docs @ v0.2.0](https://github.com/shipwright-io/build/tree/v0.2.0/docs) | [Examples @ v0.2.0](https://github.com/shipwright-io/build/tree/v0.2.0/samples) |
-| [v0.1.1](https://github.com/shipwright-io/build/releases/tag/v0.1.1)    | [Docs @ v0.1.1](https://github.com/shipwright-io/build/tree/v0.1.1/docs) | [Examples @ v0.1.1](https://github.com/shipwright-io/build/tree/v0.1.1/samples) |
-| [v0.1.0](https://github.com/shipwright-io/build/releases/tag/v0.1.0)    | [Docs @ v0.1.0](https://github.com/shipwright-io/build/tree/v0.1.0/docs) | [Examples @ v0.1.0](https://github.com/shipwright-io/build/tree/v0.1.0/samples) |
-
-## Examples
-
-Examples of `Build` resource using the example strategies installed by default.
-
-* [`buildah`](samples/build/build_buildah_cr.yaml)
-* [`buildpacks-v3-heroku`](samples/build/build_buildpacks-v3-heroku_cr.yaml)
-* [`buildpacks-v3`](samples/build/build_buildpacks-v3_cr.yaml)
-* [`kaniko`](samples/build/build_kaniko_cr.yaml)
-* [`source-to-image`](samples/build/build_source-to-image_cr.yaml)
-
-## Try it!
-
-* Get a [Kubernetes](https://kubernetes.io/) cluster and [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/) set up to connect to your cluster.
-* Clone this repository from GitHub at the v0.3.0 tag:
+* We also require a Tekton installation (v1.19+). To install the latest version, run:
 
   ```bash
-  $ git clone --branch v0.3.0 https://github.com/shipwright-io/build.git
-  ...
-  $ cd build/
+  $ kubectl apply --filename https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.20.1/release.yaml
```

Comment:
Nit: Should we align the version mentioned in the text with the one in the `kubectl`?

Reply:
you mean align v1.19 with the one in the `kubectl` cmd?

Reply:
v1.19+ was basically the same as "our oldest version is v1.19+" , so it shouldn't match what we use as our default(_v0.20.*_). The problem is that we do not really have a good way to know whats our oldest supported version. I will modify this at least to be a v1.18+, is that ok for you?

I also layout the above problem in https://github.com/shipwright-io/build/issues/681

Reply:
Updated, closing.